keyboards/cxt_studio/readme.md

```diff
@@ -0,0 +1,27 @@
+# cxt_studio
+
+![cxt_studio](https://gist.githubusercontent.com/ColinKinloch/cb49a8b42fcd07445232a26f562f3b9d/raw/ff9ac62884d724811f634c07e44c737a148181cc/cxt_studio.jpg)
```

Comment:
This image should be rehosted on Imgur.

Reply:
I can't seem to create an account. If the issue is the risk of me modifying the image then I could change it to "https://user-images.githubusercontent.com/334272/211168668-84c1cc8c-e5c5-45e7-a703-2fd47007131e.jpg" which is generated from embedding the image in this comment.
![Screenshot 2023-01-07 at 20-08-16 Imgur](https://user-images.githubusercontent.com/334272/211168639-8360665b-548f-48a8-a1ff-137d487b14db.png)
![cxt_studio](https://user-images.githubusercontent.com/334272/211168668-84c1cc8c-e5c5-45e7-a703-2fd47007131e.jpg)

Reply:
Or you could upload it?

Reply:
Yes:
`![cxt_studio](https://i.imgur.com/AMCTioSh.jpeg)`